rdtools/analysis.py

```diff
@@ -207,26 +203,38 @@ def calc_cell_temperature(self, poa, windspeed, ambient_temperature):
         numeric
             calculated cell temperature
         '''
-        if self.temperature_model is None:
+        from packaging import version
+        
+        if version.parse(pvlib.__version__) < version.parse('0.7'):
```

Comment:
#170 bumped our pvlib requirement to >=0.7.0, so I think this tapdancing around for the old pvlib API isn't necessary.

Reply:
Thanks.  I have it in here for the time being because pvfleets_qa is still on pvlib 0.6.3, but I agree that this isn't strictly necessary based on our requirements.

Reply:
I removed this and replaced with exception handling for AttributeError (no python module named 'temperature') which is raised for pvlib < 0.7